src/init_test.py

```diff
@@ -0,0 +1,21 @@
+import build
+import init
+import os
```

Comment:
```suggestion
from pathlib import Path
```

Reply:
This has been implemented in latest version